pkgs/applications/terminal-emulators/wezterm/default.nix

```diff
@@ -45,6 +46,7 @@ rustPlatform.buildRustPackage rec {
   cargoSha256 = "sha256-yjTrWoqIKoRV4oZQ0mfTGrIGmm89AaKJd16WL1Ozhnw=";
 
   nativeBuildInputs = [
+    perl
```

Comment:
If we only need perl on darwin which I think is the case since I never needed it when building for linux, please add it only conditional when we build on darwin.